src/utils/carInputValidation.js

```diff
@@ -0,0 +1,26 @@
+import { RULES } from "../constants/messages.js";
+import { ERROR_MESSAGES } from "../constants/messages.js";
+
+const carInputValidation = (CARS_NAMES) => {
+  if (CARS_NAMES === "") {
+    throw new Error(ERROR_MESSAGES.CAR_ERROR_MESSAGES.BLANK);
+  }
```

Comment:
```suggestion
  if (CARS_NAMES === "") {
    throw new Error(ERROR_MESSAGES.CAR_ERROR_MESSAGES.BLANK);
  }
```
와 빈문자열은 생각 못했던 것 같네요 많이 배워갑니다!!